src/main/java/com/example/rets_api/dto/ViewDataDTO.java

```diff
@@ -0,0 +1,29 @@
+package com.example.rets_api.dto;
+
+import com.example.rets_api.resource.Enums.*;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
```

Comment:
Now the DTO need to have @AllArgsConstructor and @NoArgsConstructor because we changed the getByFilter call and now those constructors needed to be used